receiver/scraper/helper/config.go

```diff
@@ -1,7 +1,7 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package scraperhelper // import "go.opentelemetry.io/collector/receiver/scraperhelper"
```

Comment:
Same here, I would keep this as `scraperhelper`, the repetition with the parent folder is fine